.github/actions/install-oppia-dependencies/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Install Dependencies'
+description: 'Installs wheel and third-party python libs.'
+runs:
+  using: 'composite'
+  steps:
+    - name: Setup python by installing wheel
+      run: pip install wheel==0.35.0
+      shell: bash
+    - name: Check Yarn Cache
+      run: |
+        pwd
+        cd ..
+        pwd
+        echo 'cache-folder "yarn_cache"' > .yarnrc
+        yarn cache list
+        rm .yarnrc
```

Comment:
why removing this file?

Reply:
See my reply above. This isn't the `.yarnrc` file in the Oppia repo. We are removing the new `.yarnrc` that we created two lines previously. We only needed it to set the cache folder, so now that it's not needed anymore, we delete it.